Server/mods/deathmatch/logic/lua/CLuaModule.cpp

```diff
@@ -297,59 +292,56 @@ bool CLuaModule::RegisterFunction(lua_State* luaVM, const char* szFunctionName,
     return true;
 }
 
-bool CLuaModule::GetResourceName(lua_State* luaVM, std::string& strName)
+std::string CLuaModule::GetResourceName(lua_State* luaVM)
```

Comment:
are you sure it wont break modules? as you can see in ILuaModuleManager interface class, this function has signature:
```cpp
virtual bool                GetResourceName         ( lua_State * luaVM, std::string &strName ) = 0;
```

Reply:
> are you sure it wont break modules? as you can see in ILuaModuleManager interface class, this function has signature:
> 
> ```c++
> virtual bool                GetResourceName         ( lua_State * luaVM, std::string &strName ) = 0;
> ```

It will definitely break modules. It also breaks the ABI.

Reply:
> are you sure it wont break modules? as you can see in ILuaModuleManager interface class, this function has signature:
> 
> ```c++
> virtual bool                GetResourceName         ( lua_State * luaVM, std::string &strName ) = 0;
> ```

yes it will break other modules